app/staticfiles/css/main.css

```diff
@@ -439,6 +439,31 @@ figure {
   border: 0;
 }
 
+.comment-icon {
+  width: 16px;
+  height: 16px;
+  margin-right: 0.5rem;
```

Comment:
I think if you set the color to var(--secondary-color) it will fix the dark mode issue. not sure...

Reply:
It doesn't make any difference. I'd try other solutions